cmd/cluster/create.go

```diff
@@ -104,9 +109,12 @@ func NewCreateCommand() *cobra.Command {
 	cmd.Flags().StringArrayVar(&opts.Annotations, "annotations", opts.Annotations, "Annotations to apply to the hostedcluster (key=value). Can be specified multiple times.")
 	cmd.Flags().StringVar(&opts.NetworkType, "network-type", opts.NetworkType, "Enum specifying the cluster SDN provider. Supports either Calico or OpenshiftSDN.")
 	cmd.Flags().BoolVar(&opts.FIPS, "fips", opts.FIPS, "Enables FIPS mode for nodes in the cluster")
+	cmd.Flags().StringVar(&opts.Platform, "platform", opts.Platform, "The platform to use for the cluster. Currently only supports 'aws' and 'none' for baremetal")
```

Comment:
It does not necessarily need to be for baremetal. Let's avoid the term baremetal for now until we have a clear roadmap for all the uses cases and stories in baremetal environments.